.github/workflows/Tests.yml

```diff
@@ -60,6 +60,15 @@ jobs:
       - Linux
       - X64
 
+    strategy:
+      fail-fast: false
+      matrix:
+        install:
+          - xc7
+          - ql
+
+    name: ${{ matrix.install }}
```

Comment:
Please use `name: install-${{ matrix.install }}`. There is another job named xc7 in the Test matrix, and it might be confusing.

Reply:
I edited the whole job name inside `install` matrix. Is that ok?